libs/air-discount-scheme/types/src/lib/discount.ts

```diff
@@ -6,6 +8,7 @@ export type ConnectionDiscountCode = {
 }
 
 export interface Discount {
+  user: User
```

Comment:
Note for reviewers:
The user is already cached within Discount so no change needed for that in this PR. 
The models were updated some time ago but then the type was forgotten.